### PR TITLE
Haar 3147 improve service list screen

### PR DESCRIPTION
**Pull request by thomasridd**
image

### CODE REVIEW

File: server/config.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ export interface ApiConfig {
 export default {
   buildNumber: get('BUILD_NUMBER', '1_0_0', requiredInProduction),
   productId: get('PRODUCT_ID', 'UNASSIGNED', requiredInProduction),
+  serviceAccountName: get('SERVICE_ACCOUNT_NAME', 'hmpps-subject-access-request', requiredInProduction),
```
Comment (simon-mitchell) [inline server/config.ts:35]: Don't think it needs this in

File: server/data/serviceCatalogue/serviceCatalogue.yaml [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+      label: Offender Case Notes
+      url: https://dev.offender-case-notes.service.justice.gov.uk
+      order: 22
+      disabled: false
+    - name: offender-management-allocation-manager
+      label: Offender Management Allocation Manager
+      url: https://test.moic.service.justice.gov.uk
+      order: 23
+      disabled: false
+  - environment: preprod
```
Comment (simon-mitchell) [inline server/data/serviceCatalogue/serviceCatalogue.yaml:119]: would have thought the environment(s) could be part of the group eg
dev: enabled
preprod: disabled
prod: disable

File: server/data/serviceCatalogueData.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import fs from 'fs'
+import yaml from 'js-yaml'
+import config from '../config'
+
+export interface ServiceCatalogueItem {
```
Comment (simon-mitchell) [inline server/data/serviceCatalogueData.ts:5]: Is ServiceCatalogue the correct name for this?

File: server/data/serviceCatalogue/serviceCatalogue.yaml [hunk 4]
```diff
@@ -0,0 +1,352 @@
+environments:
+  - environment: dev
+    services:
+    - name: create-and-vary-a-licence-api
+      label: Create And Vary A Licence
+      url: https://create-and-vary-a-licence-api-dev.hmpps.service.justice.gov.uk
```
[the remark this answers is not in the document]
Comment (thomasridd) [reply server/data/serviceCatalogue/serviceCatalogue.yaml:6]: this is the only point the urls enter the system

File: server/data/serviceCatalogue/serviceCatalogue.yaml (hunk 2 above)
Comment (thomasridd) [reply server/data/serviceCatalogue/serviceCatalogue.yaml:119]: this file is the only source of service urls and there are different urls for each service

File: server/data/serviceCatalogue/serviceCatalogue.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (simon-mitchell) [reply server/data/serviceCatalogue/serviceCatalogue.yaml:6]: I was wrong the URLs are needed 
I assumed the API would get the urls for the worker

**Review by simon-mitchell — APPROVED** [review]
LGTM

**Pull request merged**